### ISSUE TITLE
Extracting Communication Networks (E3) - Danescu-Niculescu-Mizil...& Potts 2013

### ISSUE THREAD

**Issue by HyunkuKwon** [UChicago-CCA-2021/Readings-Responses#21]
Post questions about the following exemplary reading here:

Danescu-Niculescu-Mizil, C., West, R., Jurafsky, D., Leskovec, J. and Potts, C., 2013. “[No country for old members: User lifecycle and linguistic change in online communities.](https://dl.acm.org/doi/10.1145/2488388.2488416)” In Proceedings of the 22nd international conference on World Wide Web: 307-318.

**Comment by lilygrier** [comment]
This article presented a unique perspective on how community-level linguistic evolution trickles down into individual-level linguistic evolution. I found it especially interesting that the early people hit linguistic adolescence and start conforming, the more likely they are to leave the platform early, as intuition would have it that early eagerness to conform may indicate larger desire to fit in and perhaps a higher valuation of the group. The world of internet forums has evolved a lot in the last decade since this article was written; I feel like I have to learn "what the kids are saying" on a very regular basis. I'm wondering how these trends operate on larger platforms like Reddit and Twitter (especially where people get to choose who to follow). Does the greater frequency with which people engage in online platforms hasten the evolution of language? If so, when people reach linguistic stability, are they likely to leave the platform or create a new subgroup within it of fellow "linguistic conservatives?"

**Comment by Bin-ary-Li** [comment]
I think this is a very interesting and elegant work. I found most of the findings to be original and convincing. However, speaking from the psycholinguistic perspective, I don’t fully agree with authors’ claim that “Our observations thus suggest that biological explanations are probably not the main source of adult language stability.” I think the user life-span on an online forum is not a good analogy for the biological life-span of a human. There are many reasons why a user will quit posting on a forum; for all it’s worth, user might just not interesting in the topic anymore. And these reasons can influence the linguistic progression and user behavior in a very conceivable way, but they are less relevant in the study of language in real life. Ultimately, the case of language usage of a online persona is not very comparable to that of a human throughout biological life-span.

**Comment by egemenpamukcu** [comment]
This was a very interesting, clear and well-thought-out paper. What I found to be most creative and impressive were the features selected by the authors. It went well beyond the conventional tf-idf or frequency vectors. Are the features used in the paper grounded in sociocology or linguistics literature? I am imagining subject matter expertise probably plays a huge role in here, because this is something different than selecting the best subset of features from a pool. What would be a good approach to thinking of what features to 'generate' in a project?

![image](https://user-images.githubusercontent.com/72174414/107718644-03d63500-6c9c-11eb-8e3b-b5affcde98cd.png)

**Comment by theoevans1** [comment]
These beer discussion platforms are interesting in that they're both very focused subcultures. That differs from platforms like Twitter or Facebook, which are more diffusive and less cohesive, and I would expect also less tight-knit. To what extent do these results hold across larger-scale platforms like those? I'm especially interested in how the linguistic behavior of different communities on the same platform might interact. On a platform like Twitter, is it possible that a lifecycle would involve not leaving the platform, but moving from one sub-community to another?

**Comment by yushiouwillylin** [comment]
The finding presented in this paper seems very interesting and have very practical methodology that can be used in online community management. I'm wondering if James know any real usage of this method in real-world setting, and how they were used to lock-in the originally leaving individuals. Furthermore, what are some of the real-life content analysis cases that really boosted business or online community's performance?

**Comment by romanticmonkey** [comment]
Would opinion leaders in certain communities have effect on how term usage changes in a community?

In addition, instead of the life cycle from old to new, would it also be interesting to investigate how "retro" terms evolves?

**Comment by xxicheng** [comment]
I agree with @Bin-ary-Li 's comments about the diverse reasons for quitting posts. There may be other confounding factors. 

Also, how should we view the language usage changes on the platform? Do they align with posters' offline term usage? Are they long term changes?

**Comment by k-partha** [comment]
These online communities might be a microcosm for society at large - assimilation of societal attitudes and cultural tokens by young adults and the subsequent ossification of these ideas as the generations age strikes me as a process paralleling these findings. 

To what extent can we draw generalizations from studies of online communities to society at large? (not just to other similar online communities?)

**Comment by Raychanan** [comment]
The authors used text from two beer rating communities. I think it's still a relatively niche community. I think in a niche community people would be more inclined to speak in jargon. So I doubt that this linguistic lifecycle could also be found in larger communities (like twitter and Reddit).

Secondly, the quantitative model used in this paper is clearly not performing as well as it should, but why did they insist on doing it anyway? Is there no better way to do it?

**Comment by Rui-echo-Pan** [comment]
I have a similar quesiton with @egemenpamukcu , how could we decide the general direction of features to explore, and the more precise definition of features? Is it more based on theory and assumption, or the data-driven method itself?

**Comment by jinfei1125** [comment]
The findings of this paper definitely exist in our online environment and I am amazed that researchers can find such interesting and scientific study about our daily normal life.  But I am a little aware of the number of members is increasing across time, could it be a confounder?
Also, I am also curious about @Raychanan 's question, are there any better alternative models?

**Comment by ming-cui** [comment]
I am curious if there are companies using computational tools to analyze their employees' emails. In doing so, the companies may adjust their training, mentoring, promoting, etc. But is it unethical? Especially, specific individuals will be targeted in this process.

**Comment by sabinahartnett** [comment]
I found this paper and idea really interesting and would be excited about work that extends this exploration beyond the subculture of beer-enthusiasts :) 
I am wondering, similar to Ray, whether these trends exist in communities with fewer shared interests and thus less common jargon? In that way, this study feels like a great way to test out the methods they imposed, perhaps for exploration on larger, more diverse corpora.

**Comment by mingtao-gao** [comment]
I found this paper did an excellent job in describing the linguistic changes. It shows users' language become more conservative after linguistic adolescence. My question is does such a result turn to other communities? For example, the group who look at the post much more frequently?
